docs/data/sdks/ios-swift/migration.md

```diff
@@ -0,0 +1,513 @@
+---
+title: iOS SDK Migration Guide
+description: Use this guide to easily migrate from Amplitude's legacy iOS SDK (Amplitude-iOS) to the new SDK (Amplitude-Swift).
+---
+
+The new version of Amplitude's iOS SDK (`Amplitude-Swift`) features a plugin architecture, built-in type definition and broader support for front-end frameworks. The new version isn't backwards compatible with `Amplitude-iOS`. 
+
+To migrate to `Amplitude-Swift`, update your dependencies and instrumentation.
+
+## Terminology
+
+* `Amplitude-iOS`: Legacy iOS SDK
+* `Amplitude-Swift`: New iOS SDK
+
+## Dependency
+
+For CocoaPods installation:
+=== "Amplitude-iOS"
+    Add dependency to Podfile.
+    ```
+    pod 'Amplitude', '~> 8.8.0'
+    ```
+
+=== "Amplitude-Swift"
+    Add dependency to Podfile.
+    ```
+    pod 'AmplitudeSwift', '~> 0.3.0'
```

Comment:
Albeit confusing, I feel like we should name the new pod "AmplitudeIOS" or "AmplitudeAnalytics" to avoid confusion on supported languages like we did with the "TypeScript" SDK (i.e. does this work for ObjC?)

Reply:
Thanks for the review. Yea, I think that's a good suggestion. I like the AmplitudeAnalytics more. But we already have a release before. Is there any concern if we change the package name now?

Reply:
Merge this. And will have offline discussion related to updating the library name.